### PR TITLE
refactor(text-input): implement design tokens spec

### PR DESCRIPTION
**Pull request by badejayayesubabu**
## Description

`<text-input>` : implement design tokens styles

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2203

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/scss/components/text-input/_index.scss [hunk 1]
```diff
@@ -138,6 +138,7 @@ hx-text-control {
 
   > p.hxErrorText {
     color: $red-status-500;
+    color: $rax-component-input-error-border-bottom-color;
```
Comment (100stacks) [inline src/scss/components/text-input/_index.scss:141]: Please update to use IE11 fallback pattern:

```scss
color: $rax-component-input-error-border-bottom-color;
color: var(--hxTextControl-hxErrorText, $rax-component-input-error-border-bottom-color);
```

**Review by 100stacks — CHANGES_REQUESTED** [review]
### @badejayayesubabu, missing numerous design tokens.  Please review file for values.

File: src/scss/components/text-input/_index.scss (hunk 1 above)
Comment (badejayayesubabu) [reply src/scss/components/text-input/_index.scss:140]: Changes have been done, Please review

File: src/scss/components/text-input/_index.scss [hunk 2]
```diff
@@ -137,9 +137,10 @@ hx-text-control {
   }
 
   > p.hxErrorText {
```
Comment (100stacks) [inline src/scss/components/text-input/_index.scss:139]: Please review file for additional design token usage.  See spacing tokens.  

https://github.com/HelixDesignSystem/helix-ui/blob/b6b0f42957c6eae9025693aab2250f6fea09d485/src/scss/components/text-input/_index.scss#L65-L92

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/text-input/_index.scss (hunk 2 above)
Comment (badejayayesubabu) [reply src/scss/components/text-input/_index.scss:139]: Changes have been done, Please review

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**